WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -56,6 +56,7 @@
             <% end %>
             <li class="divider"></li>
             <li><a href="/results/statistics.php"><%= icon('area-chart') %> <%= t '.statistics' %></a></li>
+            <li><a href="<%= relations_path %>"><%= icon('sitemap') %> <%= t '.relations' %></a></li>
```

Comment:
Sorry for being a party killer, but I don't think this should be linked in the navigation bar.
People already have problems to understand the navigation elements and find what they are looking for, so adding another link, which has no relevance for the random visitor, doesn't sound like a good idea.

Nitpick: this is also the same icon as the icon we use for delegates in the "Information" submenu.